### PR TITLE
Replaced legacy babel-* & shorthand package name usage with @babel/* in README.md's

### PR DESCRIPTION
**Pull request by hulkish**
| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            | N/A
| Patch: Bug Fix?          | N/A
| Major: Breaking Change?  | N/A
| Minor: New Feature?      | N/A
| Tests Added + Pass?      | No + Yes
| Documentation PR         | Yes
| Any Dependency Changes?  | No
| License                  | MIT

### CODE REVIEW

File: experimental/babel-preset-env/CONTRIBUTING.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ And add them in this structure:
 
 ```js
 // es2016
-"@babel/transform-exponentiation-operator": {
```
Comment (hzoo) [inline experimental/babel-preset-env/CONTRIBUTING.md]: actually this whole document is kind of out of date

https://github.com/hulkish/babel/blob/master/experimental/babel-preset-env/data/plugin-features.js, referencing the old way to do things + wrong links

**Comment by g0t4** [comment]
Is the shorthand format still valid to use? Or is it deprecated/removed in v7?

**Comment by existentialism** [comment]
@g0t4 it's still valid for v7, just normalizing our docs to not use it since it's confused many people

File: experimental/babel-preset-env/CONTRIBUTING.md (hunk 1 above)
Comment (hulkish) [reply experimental/babel-preset-env/CONTRIBUTING.md]: This link is fixed 👍

File: experimental/babel-preset-env/README.md [hunk 2]
```diff
@@ -235,15 +235,15 @@ An array of plugins to always include.
 
 Valid options include any:
 
-- [Babel plugins](https://github.com/babel/babel-preset-env/blob/master/data/plugin-features.js) - both with (`@babel/plugin-transform-spread`) and without prefix (`transform-spread`) are supported.
+- [Babel plugins](https://github.com/babel/babel-preset-env/blob/master/data/plugin-features.js) - both with (`@babel/plugin-transform-spread`) and without prefix (`@babel/plugin-transform-spread`) are supported.
```
Comment (existentialism) [inline experimental/babel-preset-env/README.md]: As @hzoo pointed out earlier, the include/exclude options in `preset-env` still need to get updated (kinda think we should do this in a separate PR? ¯\_(ツ)_/¯)

Relevant files:
[index.js#L188](https://github.com/babel/babel/blob/master/experimental/babel-preset-env/src/index.js#L188)
[available-plugins.js](https://github.com/babel/babel/blob/master/experimental/babel-preset-env/src/available-plugins.js)
[plugin-features.js](https://github.com/babel/babel/blob/master/experimental/babel-preset-env/data/plugin-features.js)

Also, the second shouldn't have a prefix as it's describing that shorthand is supported when declaring include/excludes.

Comment (hzoo) [reply experimental/babel-preset-env/README.md]: should be https://github.com/babel/babel/pull/6825

**Comment by hulkish** [comment]
@existentialism @g0t4 @hzoo alright how's that, guys?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -161,16 +161,16 @@ In case you're locally getting errors which are not on the CI, it may be due to
 
 Most packages in [`/packages`](https://github.com/babel/babel/tree/master/packages) have a `test` folder, however some tests might be in other packages or in [`/packages/babel-core`](https://github.com/babel/babel/tree/master/packages/babel-core/test/fixtures).
 
-#### `babel-plugin-x`
+#### `@babel/plugin-x`
 
 All the Babel plugins (and other packages) that have a `/test/fixtures` are written in a similar way.
 
-For example, in [`babel-plugin-transform-exponentiation-operator/test`](https://github.com/babel/babel/tree/master/packages/babel-plugin-transform-exponentiation-operator/test):
+For example, in [`@babel/plugin-transform-exponentiation-operator/test`](https://github.com/babel/babel/tree/master/packages/babel-plugin-transform-exponentiation-operator/test):
```
Comment (hzoo) [inline CONTRIBUTING.md]: Not sure where to change these because the folder names are still using `-` but I guess this is fine.

**Comment by hzoo** [comment]
thanks @hulkish !

**Pull request merged**